<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MannagerForm</class>
 <widget class="QWidget" name="MannagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智慧小区</string>
  </property>
  <widget class="QLabel" name="h_lable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="h_btn1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="h_btn2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_btn1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
